src/Scaffold_Command.php

```diff
@@ -201,6 +200,106 @@ private function _scaffold( $slug, $assoc_args, $defaults, $subdir, $templates )
 		}
 	}
 
+	/**
+	 * Generate PHP, JS and CSS code for registering a custom block.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <slug>
+	 * : The internal name of the block.
+	 *
+	 * [--title=<title>]
+	 * : The display title for your block.
+	 *
+	 * [--dashicon=<dashicon>]
+	 * : The dashicon to make it easier to identify your block.
+	 *
+	 * [--category=<category>]
+	 * : The category name to help users browse and discover your block.
+	 *
+	 * [--textdomain=<textdomain>]
+	 * : The textdomain to use for the labels.
+	 *
+	 * [--theme]
+	 * : Create files in the active theme directory. Specify a theme
+	 * with `--theme=<theme>` to have the file placed in that theme.
```

Comment:
I'd advocate for not including a `theme` parameter here. Blocks are content and probably shouldn't stop being supported when you switch themes.

Reply:
I wasn't sure if include theme option here, but I decided to do it based on the [Block API documentation](https://wordpress.org/gutenberg/handbook/block-api/) which includes the following:
> Blocks are the fundamental element of the Gutenberg editor. They are the primary way in which plugins and themes can register their own functionality and extend the capabilities of the editor.

@mtias, what is your opinion about it?